cafebabel/articles/models.py

```diff
@@ -2,14 +2,33 @@
 
 from flask import current_app, url_for
 from flask_login import current_user
-from mongoengine import signals
+from mongoengine import PULL, signals
 
 from .. import db
 from ..core.helpers import slugify
 from ..core.mixins import UploadableImageMixin
 from ..users.models import User
 
 
+class Tag(db.Document, UploadableImageMixin):
```

Comment:
`UploadableImageMixin` requires the implementation of `get_images_url()` and `get_images_url()`

Reply:
Done.